### PR TITLE
Adding question and solution for perform string shift

### PR DESCRIPTION
**Pull request by mvignesh18**
PR for string shift prob

### CODE REVIEW

**Review by sooriyan90 — COMMENTED** [review]
This is not a stand alone java code. You didn’t import the package. There’s no void main to execute your code. Make the pr in such a way, that you can run the code.

**Comment by sooriyan90** [comment]
Using the second hint, I think you can simplify the rotate to only once. And also using a string builder/ buffer

**Comment by mvignesh18** [comment]
Included the package and main method - now you should be able to run it.

**Comment by mvignesh18** [comment]
> Using the second hint, I think you can simplify the rotate to only once. And also using a string builder/ buffer

Yes. My second solution does the rotate only once.

**Comment by sooriyan90** [comment]
Why are there 2 different files here?

File: src/Strings/PerformStringShift.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
+/** QUESTION
```
Comment (sooriyan90) [inline src/Strings/PerformStringShift.java:1]: This file is not needed, delete it and push again

Comment (mvignesh18) [reply src/Strings/PerformStringShift.java:1]: This was the original file I commited. @sekhnar can you place your solution in this file?

**Review by sooriyan90 — APPROVED** [review]

**Pull request merged**